wiki/Glossary/Combo_fire/en.md

```diff
@@ -1,15 +1,16 @@
 ---
-stub: true
 tags:
   - combo flame
 ---
 
 # Combo fire
 
-![Ancient screenshot of the combo fire](/wiki/shared/combo-fire.jpg "R.I.P. combo fire")
+<!-- TODO(TicClick): wrap the images in infoboxes -->
 
-**Combo fire** was a background gameplay element that originated from the [osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia") DS Game that eventually got integrated into osu!. This would display a burning, yellow flame after obtaining a [combo milestone](/wiki/Glossary/Combo_milestone). If the player obtained a [combo](/wiki/Glossary/Combo_(score_multiplier)) of 500x, the combo fire would change colour from yellow to blue. The fire would extinguish if the combo was broken, but would be reobtainable if combo was regained.
+![](img/combo-fire.jpg "Yellow, orange, and red combo fire (below 500x combo)")
 
-Combo fire was disabled on March 5, 2013 due to performance concerns.
+![](img/blue-combo-fire.jpg "Blue combo fire (above 500x combo)")
+
+**Combo fire** was a background gameplay element that originated from the [Osu! Tatakae! Ouendan!](/wiki/iNiS_games) DS Game that eventually got integrated into osu!. This would display a burning, yellow flame after obtaining the first [combo milestone](/wiki/Glossary/Combo_milestone). If the player obtained a [combo](/wiki/Glossary/Combo_(score_multiplier)) of 500x, the combo fire would change colour from yellow to blue. The fire would extinguish if the combo was broken, but would be reobtainable if combo was regained.
```

Comment:
```suggestion
**Combo fire** was a background gameplay element that originated from the [Osu! Tatakae! Ouendan!](/wiki/iNiS_games) DS game, and eventually got integrated into osu!. This would display a burning, yellow flame after obtaining the first [combo milestone](/wiki/Glossary/Combo_milestone). If the player obtained a [combo](/wiki/Glossary/Combo_(score_multiplier)) of 500, the combo fire would change colour from yellow to blue. The fire would extinguish if the combo was broken, but it would reappear if combo was regained.
```